api/primitives/alloc-only.txt

```diff
@@ -1079,24 +1079,40 @@ pub const fn bitcoin_primitives::WitnessMerkleNode::to_byte_array(self) -> [u8;
 pub const fn bitcoin_primitives::Wtxid::as_byte_array(&self) -> &[u8; 32]
 pub const fn bitcoin_primitives::Wtxid::from_byte_array(bytes: [u8; 32]) -> Self
 pub const fn bitcoin_primitives::Wtxid::to_byte_array(self) -> [u8; 32]
+pub const fn bitcoin_primitives::block::Block<bitcoin_primitives::block::Checked>::cached_witness_root(&self) -> core::option::Option<bitcoin_primitives::WitnessMerkleNode>
+pub const fn bitcoin_primitives::block::Block<bitcoin_primitives::block::Checked>::header(&self) -> &bitcoin_primitives::block::Header
+pub const fn bitcoin_primitives::block::BlockHashDecoder::new() -> Self
 pub const fn bitcoin_primitives::block::Version::from_consensus(v: i32) -> Self
+pub const fn bitcoin_primitives::block::Version::to_consensus(self) -> i32
+pub const fn bitcoin_primitives::block::VersionDecoder::new() -> Self
+pub const fn bitcoin_primitives::pow::CompactTarget::to_consensus(self) -> u32
+pub const fn bitcoin_primitives::pow::CompactTargetDecoder::new() -> Self
+pub const fn bitcoin_primitives::script::RedeemScriptSizeError::invalid_size(&self) -> usize
 pub const fn bitcoin_primitives::script::Script<T>::as_bytes(&self) -> &[u8]
 pub const fn bitcoin_primitives::script::Script<T>::from_bytes(bytes: &[u8]) -> &Self
 pub const fn bitcoin_primitives::script::Script<T>::is_empty(&self) -> bool
 pub const fn bitcoin_primitives::script::Script<T>::len(&self) -> usize
 pub const fn bitcoin_primitives::script::Script<T>::new() -> &'static Self
 pub const fn bitcoin_primitives::script::ScriptBuf<T>::from_bytes(bytes: alloc::vec::Vec<u8>) -> Self
 pub const fn bitcoin_primitives::script::ScriptBuf<T>::new() -> Self
+pub const fn bitcoin_primitives::script::ScriptBufDecoder<T>::new() -> Self
 pub const fn bitcoin_primitives::script::ScriptHash::as_byte_array(&self) -> &[u8; 20]
 pub const fn bitcoin_primitives::script::ScriptHash::from_byte_array(bytes: [u8; 20]) -> Self
 pub const fn bitcoin_primitives::script::ScriptHash::to_byte_array(self) -> [u8; 20]
 pub const fn bitcoin_primitives::script::WScriptHash::as_byte_array(&self) -> &[u8; 32]
 pub const fn bitcoin_primitives::script::WScriptHash::from_byte_array(bytes: [u8; 32]) -> Self
 pub const fn bitcoin_primitives::script::WScriptHash::to_byte_array(self) -> [u8; 32]
+pub const fn bitcoin_primitives::script::WitnessScriptSizeError::invalid_size(&self) -> usize
```

Comment:
I don't rekon we should do errors. It doesn't add much value and it restricts the forward maintainability. I can't think of a reason one would need to get the error inners in const context?